examples/step-22/doc/intro.dox

```diff
@@ -818,8 +818,9 @@ shows how the profile looked at an early stage of optimizing this program:
 
 <img src="https://www.dealii.org/images/steps/developer/step-22.profile-3.original.png" alt="">
 
-As mentioned above, the runtime of this version was about four times as long
-as for the first profile, with the SparseILU decomposition taking up about
-30% of the instruction count, and operations on the ill-suited
-CompressedSparsityPattern about 10%. Both these bottlenecks have since been
+As mentioned above, the runtime of this version was about four times as long as
+for the first profile, with the SparseILU decomposition taking up about 30% of
+the instruction count, and operations on the ill-suited
+CompressedSparsityPattern (which, as of early 2015, has been replaced by
+DynamicSparsityPattern) about 10%. Both these bottlenecks have since been
```

Comment:
I added that current note because the kcachegrind picture (on the website) shows the CompressedSparsityPattern. I updated it to not mention the old name.